src/benchmarks/gc/GC.Infrastructure/Notebooks/Examples/CustomTraceEventExample.ipynb

```diff
@@ -1,304 +0,0 @@
-{
- "cells": [
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Usage of GC.Analysis.API for GC Analysis"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "dotnet_interactive": {
-     "language": "csharp"
-    },
-    "polyglot_notebook": {
-     "kernelName": "csharp"
-    },
-    "vscode": {
-     "languageId": "polyglot-notebook"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "// In order to use a custom TraceEvent library, we need to reference it directly here\n",
-    "\n",
-    "#r \"C:\\Dev\\perfview\\src\\TraceEvent\\bin\\Release\\netstandard2.0\\Microsoft.Diagnostics.Tracing.TraceEvent.dll\""
```

Comment:
Do we have to delete the CustomTraceEvent example?
I think we will have times where we wanted to perform our analysis with a custom trace event (e.g. to fix a bug in TraceEvent or otherwise)

Reply:
We didn't delete but rename the notebook as "CustomDynamicEvents".